orchestra/communication/slack.py

```diff
@@ -29,6 +32,12 @@ def _silent_request(*args, **kwargs):
         return _request(*args, **kwargs)
     except SlackError:
         logger.exception('Slack API Error')
+    except HTTPError as e:
+        status_code = e.response.status_code
+        if status_code == 429:
+            logger.exception('HTTPError: {}'.format(e))
+        else:
+            raise(e)
```

Comment:
you can re-raise so you don't lose the information in the original exception: http://www.ianbicking.org/blog/2007/09/re-raising-exceptions.html

```suggestion
            raise
```